src/mscorlib/src/System/Runtime/Intrinsics/X86/Sse2.cs

```diff
@@ -14,858 +14,870 @@ namespace System.Runtime.Intrinsics.X86
+        /// __m128d _mm_max_pd (__m128d a,  __m128d b); MAXPD xmm, xmm/m128
         /// </summary>
         public static Vector128<double> Max(Vector128<double> left, Vector128<double> right) => Max(left, right);
 
         /// <summary>
-        /// __m128d _mm_max_sd (__m128d a,  __m128d b)
+        /// __m128d _mm_max_sd (__m128d a,  __m128d b); MAXSD xmm, xmm/m64
         /// </summary>
         public static Vector128<double> MaxScalar(Vector128<double> left, Vector128<double> right) => MaxScalar(left, right);
 
         /// <summary>
-        /// __m128i _mm_min_epu8 (__m128i a,  __m128i b)
+        /// __m128i _mm_min_epu8 (__m128i a,  __m128i b); PMINUB xmm, xmm/m128
         /// </summary>
         public static Vector128<byte> Min(Vector128<byte> left, Vector128<byte> right) => Min(left, right);
         /// <summary>
-        /// __m128i _mm_min_epi16 (__m128i a,  __m128i b)
+        /// __m128i _mm_min_epi16 (__m128i a,  __m128i b); PMINSW xmm, xmm/m128
         /// </summary>
         public static Vector128<short> Min(Vector128<short> left, Vector128<short> right) => Min(left, right);
         /// <summary>
-        /// __m128d _mm_min_pd (__m128d a,  __m128d b)
+        /// __m128d _mm_min_pd (__m128d a,  __m128d b); MINPD xmm, xmm/m128
         /// </summary>
         public static Vector128<double> Min(Vector128<double> left, Vector128<double> right) => Min(left, right);
 
         /// <summary>
-        /// __m128d _mm_min_sd (__m128d a,  __m128d b)
+        /// __m128d _mm_min_sd (__m128d a,  __m128d b); MINSD xmm, xmm/m64
         /// </summary>
         public static Vector128<double> MinScalar(Vector128<double> left, Vector128<double> right) => MinScalar(left, right);
 
         /// <summary>
-        /// __m128d _mm_move_sd (__m128d a, __m128d b)
+        /// __m128d _mm_move_sd (__m128d a, __m128d b); MOVSD xmm, xmm
         /// </summary>
         public static Vector128<double> MoveScalar(Vector128<double> upper, Vector128<double> value) => MoveScalar(upper, value);
 
         /// <summary>
-        /// int _mm_movemask_epi8 (__m128i a)
+        /// int _mm_movemask_epi8 (__m128i a); PMOVMSKB reg, xmm
         /// </summary>
         public static int MoveMask(Vector128<sbyte> value) => MoveMask(value);
         /// <summary>
-        /// int _mm_movemask_pd (__m128d a)
+        /// int _mm_movemask_epi8 (__m128i a); PMOVMSKB reg, xmm
+        /// </summary>
+        public static int MoveMask(Vector128<byte> value) => MoveMask(value);
+        /// <summary>
+        /// int _mm_movemask_pd (__m128d a); MOVMSKPD reg, xmm
         /// </summary>
         public static int MoveMask(Vector128<double> value) => MoveMask(value);
 
         /// <summary>
-        /// __m128i _mm_mul_epu32 (__m128i a,  __m128i b)
+        /// __m128i _mm_move_epi64 (__m128i a); MOVQ xmm, xmm
+        /// </summary>
+        public static Vector128<long> MoveScalar(Vector128<long> value) => MoveScalar(value);
```

Comment:
What is the reasoning for this being `MoveScalar`, but changing `SetScalar` to `SetScalarVector128` when I believe in both cases the entire 128-bit vector is defined.

Reply:
The former (`MoveScalar`) takes and returns `Vector128`, so it shouldn't cause overload resolution problems. The latter (`SetScalar`) takes a primitive and returns a `Vector128`, so it could potentially conflict.

I can change if you think its clearer to provide it here as well.

Reply:
I think it would be more consistent, but I don't feel strongly about it.